README.md

```diff
@@ -1,4 +1,7 @@
-[<img src="https://travis-ci.org/sul-dlss/dor-services.png"/>](http://travis-ci.org/sul-dlss/dor-services) | [![Coverage Status](https://coveralls.io/repos/sul-dlss/dor-services/badge.svg?branch=develop&service=github)](https://coveralls.io/github/sul-dlss/dor-services?branch=develop)
+[![Build Status](https://travis-ci.org/sul-dlss/dor-services.svg?branch=master)](https://travis-ci.org/sul-dlss/dor-services)
+[![Coverage Status](https://coveralls.io/repos/sul-dlss/dor-services/badge.svg?branch=develop&service=github)](https://coveralls.io/github/sul-dlss/dor-services?branch=develop)
```

Comment:
seems like a holdover from the existing, but i'm not sure i get the `branch=develop` part?  any idea as to why not master?